### PR TITLE
Better formatting for some docstrings

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

**Comment by ilopezgp** [comment]
This will probably pass if you rebase

File: src/EnsembleKalmanProcess.jl [hunk 1]
```diff
@@ -24,15 +23,16 @@ abstract type Process end
 
 
 """
-    EnsembleKalmanProcess{FT<:AbstractFloat, IT<:Int}
+    struct EnsembleKalmanProcess{FT <: AbstractFloat, IT <: Int, P <: Process}
```
Comment (ilopezgp) [inline src/EnsembleKalmanProcess.jl:26]: Our standard convention for docstrings avoids the type (function, struct,...). You can see this in other packages of CliMA.

Comment (navidcy) [reply src/EnsembleKalmanProcess.jl]: I did that elsewhere.

File: src/EnsembleKalmanProcess.jl [hunk 2]
```diff
@@ -51,6 +51,17 @@ struct EnsembleKalmanProcess{FT <: AbstractFloat, IT <: Int, P <: Process}
...
+"""
+    EnsembleKalmanProcess(
+        params::Array{FT, 2},
+        obs_mean,
+        obs_noise_cov::Array{FT, 2},
+        process::P;
+        Δt = FT(1),
+    ) where {FT <: AbstractFloat, P <: Process}
+
+Construct an  Ensemble Kalman process.
```
Comment (ilopezgp) [inline src/EnsembleKalmanProcess.jl:63]:
```suggestion
Ensemble Kalman process constructor.
```

**Review by ilopezgp — APPROVED** [review]
LGTM, thanks! Feel free to merge after rebasing to 1 commit.

**Comment by navidcy** [comment]
OK, codecove/patch is failing but I can't see why. I didn't add any function, just rewrote it in one line.

**Pull request merged**